R/lag_antall_skjema_tabell.R

```diff
@@ -0,0 +1,263 @@
+#' @import dplyr
+#' @importFrom lubridate date
+NULL
```

Comment:
Skal vi ha en tom linje etter 'NULL' her?

Reply:
Kunne godt hatt det her, ja. Men har uansett tenkt å gå over til å ha éi sentralisert fil for import, slik me gjorde for `rapwhale`. Då vil dette forsvinna heilt frå alle funksjonsfilene.